universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -51,6 +44,11 @@ public Content render(Http.Request request, Optional<String> message) {
                     "Continue",
                     routes.CallbackController.callback(GuestClient.CLIENT_NAME).url())));
 
+    if (!message.orElse("").equals("login")) {
```

Comment:
Nit: I think we need to negate the condition here.

Reply:
Done